web/src/utils/Cryption.ts

```diff
@@ -0,0 +1,28 @@
+import CryptoJS from 'crypto-js'
+
+const keyValue:any = process.env.REACT_APP_ENCRYPTION_KEY
+const privateKeyValue:any = process.env.REACT_APP_ENCRYPTION_PRIVATE_KEY
+
+export function Encrypt(data:string) {
+    try {
```

Comment:
Why Try catch? you returning the exception mesasge as decrypted value? You know this can go wrong?